### PR TITLE
add/feat/design/reservation-component

### PR DESCRIPTION
**Pull request by AlinaHauriluk**

### CODE REVIEW

File: client/src/components/reservation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    { id: 9, time: "6 PM" },
+    { id: 10, time: "7 PM" },
+    { id: 11, time: "8 PM" },
+    { id: 12, time: "9 PM" },
+    { id: 13, time: "10 PM" },
+    { id: 14, time: "11 PM" },
+  ];
+
+  const personCount = [
+    { id: 1, person: "2 person" },
```
Comment (vladimirtopolev) [inline client/src/components/reservation/index.tsx]: Why it's a string? It's just a number of persons. Just use an input with `type=number` and set `min, max` properties

File: client/src/components/reservation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        <ReservationTitle>Make a Reservation</ReservationTitle>
+        <Form onSubmit={handleSubmit}>
+          <FlexCenter>
+            <ReservationInput
+              type="date"
+              required
+              value={inputValue}
+              onChange={(e) => setInputValue(e.target.value)}
+            />
+            <ReservationSelect value={timeValue} onChange={handleChangeTime}>
```
Comment (vladimirtopolev) [inline client/src/components/reservation/index.tsx]: What about input with `type=time` ?

File: client/src/components/reservation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  ReservationButton,
+  Form,
+} from "./reservation";
+import { FlexColumnContainer, FlexCenter } from "../../box/flexBox";
+import { BlockSubTitle } from "../../text/text";
+
+const Reservation = () => {
+  const [inputValue, setInputValue] = useState<string>("");
+  const [timeValue, setTimeValue] = useState<string>("2 PM");
+  const [personValue, setPersonValue] = useState<string>("2 person");
```
Comment (vladimirtopolev) [inline client/src/components/reservation/index.tsx]: It's the same issue like you did with the price, '2$' - it's not correct. Type should be number here, 'person' is just a unit for it.

File: client/src/components/reservation/reservation.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  font-size: ${({ theme }) => theme.fontSize.title};
+  color: ${({ theme }) => theme.colors.primary};
+  font-weight: 400;
+  line-height: 0.75;
+  margin-bottom: 60px;
+`;
+export const Form = styled.form`
+  width: 100%;
+`;
+export const ReservationInput = styled.input`
```
Comment (vladimirtopolev) [inline client/src/components/reservation/reservation.ts]: Why does input have domain area of Reservation? We won't go to use inputs with the same styles in another forms? I suppose that we should have a bunch of styled form components which will be reused across the project.

Comment (AlinaHauriluk) [reply client/src/components/reservation/reservation.ts]: I've already done it

File: client/src/components/reservation/index.tsx (hunk 3 above)
Comment (AlinaHauriluk) [reply client/src/components/reservation/index.tsx]: i understood about it

**Pull request merged**